wiki/models/urlpath.py

```diff
@@ -33,7 +33,6 @@ class URLPath(MPTTModel):
     INHERIT_PERMISSIONS = True
     
     objects = managers.URLPathManager()
-    _default_manager = objects
```

Comment:
I needed this for the 1.10 upgrade. Related to https://github.com/django-mptt/django-mptt/issues/369.